src/ui/components/WalletAvatar/WalletAvatar.tsx

```diff
@@ -18,7 +19,13 @@ export function WalletAvatar({
   icon?: React.ReactNode;
   onReady?(): void;
 }) {
-  const { data: nft, isLoading } = useProfileNft(address);
+  const { data, isLoading } = useWalletsMetaByChunks({
```

Comment:
Since this is a request for a single address, `useWalletMeta` should be used

See https://github.com/zeriontech/zerion-wallet-extension/pull/759/files#r2177227801